### PR TITLE
Portfolio with Contact form

### PR DESCRIPTION
**Pull request by djo1975**
Description
For the third milestone of your portfolio website, you will:

Add a contact form to your portfolio and the portfolio of your coding partner.
Make sure that you guide your users' experience in a way that helps them fill in the form without frustration.
IMPORTANT NOTE: Read all requirements before you start building your project.

General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
HTML/CSS requirements
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Project requirements
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

The section you need to build for this milestone is:
the contact form in the footer:
Form

We included a screenshot of Template 1 for reference, but you should follow the template that you chose.

Make sure that:
Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.
Collect data by using the [Formspree service](https://formspree.io/html).
In order to lay out the elements on the page you should use Flexbox.
You don't need to implement any functionality that requires JavaScript.
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,195 +7,157 @@
...
-
-    <section id="introduction">
-      <h1>I’m Mladan, Glad to see you!</h1>
-      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-  <a>LET'S CONNECT</a>
+  <!--Section Introduction-->
+<section id="introduction">
+  <h1 class="im">I’m Mladan, Glad to see you!</h1>
+  <p class="soft-de">I’m a software developer! I can help you<br> build a product , feature or website Look<br>through some of my work and<br>experience! If you like what you see and<br> have a project you need coded, don’t<br> hestiate to contact me.</p>
+  <a class="let-con">LET'S CONNECT</a>
```
Comment (PowerLevel9000) [inline index.html:23]: kindly use some meaningful classes so that it is easier to read the code for other developers

File: index.html [hunk 2]
```diff
@@ -7,195 +7,157 @@
...
-      <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
-      
-                <ul class="prog-lang">
-                  <li>html</li>
-                  <li>css</li>
-                  <li>javaScript</li>
-                </ul>
-                <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
-              </div>
+      <div><a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a></div></div>
```
Comment (PowerLevel9000) [inline index.html:88]: i suggest u to always use class on div because it is helpful to recognize in one glance

**Review by PowerLevel9000 — COMMENTED** [review]
#you are close 
JustThisCloseDanielIbbertsonGIF

**Comment by KAYEMBAJONATHAN** [comment]
- Make sure that your comment are meaningful  
- Your comment have to sound respectful
- And always in line 
- For example 👎 
- I would like to, may you please, can you kindly change this or that, please make sure...

File: index.html [hunk 3]
```diff
@@ -7,195 +7,157 @@
...
-      </div>
-
-      <div class="flex-item">
-        <a>LET'S CONNECT</a>
-      </div>
-
-      <div class="flex-item">
+      <div class="flex-item"><h5 class="omeni">About<br>Myself</h5></div>
+      <div class="flex-item"> <p class="intro">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+      <div class="flex-item"><a class="cone">LET'S CONNECT</a></div>
```
Comment (jkanyi-web) [inline index.html:112]: Please use correct indentation for your entire code.

**Review by jkanyi-web — CHANGES_REQUESTED** [review]

**Pull request merged**